### PR TITLE
Fix create map reducer export

### PR DESCRIPTION
**Pull request by kahummer**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,17 +17,17 @@ gisida.version = require('../package.json').version;
 gisida.initStore = require('./store/initStore').default;
 gisida.loadLayers = require('./store/initStore').loadLayers;
 gisida.reducerRegistry = require('./store/reducerRegistry').default;
-gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').default;
+gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').createMapReducer;
```
Comment (cKellyDesign) [inline src/index.js]: Why / when did this change?

File: src/index.js [hunk 2]
```diff
@@ -17,17 +17,17 @@ gisida.version = require('../package.json').version;
...
-  styles,
-  supersetConfig,
+    ...app,
+    filter,
+    layersReducer,
+    loc,
+    locationsReducer,
+    regions,
+    styles,
+    supersetConfig,
```
Comment (cKellyDesign) [inline src/index.js]: Let's keep tabbing at 2 spaces 🙏

File: src/index.js [hunk 3]
```diff
@@ -83,4 +83,4 @@ gisida.localStorage.loadState = require('./utils/localStorage').loadState;
 gisida.localStorage.saveState = require('./utils/localStorage').saveState;
 gisida.isTokenExpired = require('./utils/isTokenExpired').default;
 
-module.exports = gisida;
+module.exports = gisida;
```
Comment (cKellyDesign) [inline src/index.js]: Let's keep an extra line at end of file

File: src/map/buildDetailView.js [hunk 4]
```diff
@@ -201,7 +201,7 @@ export default (
   const layerObjDatum =
     activeData &&
     activeData.length &&
-    activeData.find(d => (d.properties || d)[join[1]] === featureProperties[join[0]]);
+    activeData.find(d => (d.properties || d)[join && join[1]] === featureProperties[join && join[0]]);
```
Comment (cKellyDesign) [inline src/map/buildDetailView.js]: Yeah that's a good catch, is there anywhere else in the codebase we're not already checking for join?

File: src/index.js (hunk 2 above)
Comment (moshthepitt) [reply src/index.js]: @kahummer for this and the other one that Conor noted below, can we add linting rule and [prettier](https://prettier.io/) (you can pair this nicely with [pre-commit](https://www.npmjs.com/package/pre-commit)) to assist us?

File: src/index.js (hunk 1 above)
Comment (kahummer) [reply src/index.js]: @cKellyDesign The reducers.js file was split into various modules which currently exist on separate files. here is the current createmapreducer https://github.com/onaio/gisida/blob/staging/src/store/reducers/createMapReducer/createMapReducer.js

File: src/map/buildDetailView.js (hunk 4 above)
Comment (kahummer) [reply src/map/buildDetailView.js]: @cKellyDesign not sure whether similar cases exist. I spotted this issue when fixing an issue on somnutrition sites

File: src/index.js (hunk 1 above)
Comment (kelvin-muchiri) [inline src/index.js]: This is nice. I noticed that one of the things I missed while refactoring the reducers was to refactor this file.

**Pull request closed without merging**